lib/y2_module_guitest.pm

```diff
@@ -60,7 +62,8 @@ sub launch_yast2_module_x11 {
     diag 'assuming root-auth-dialog, typing password';
     type_password;
     send_key 'ret';
-    assert_screen $args{target_match}, $args{match_timeout};
+    wait_still_screen 2, 2;
```

Comment:
this if needed at all, should be only inside the if.

Reply:
removed.